docs/USER_EXPERIENCE_PROPOSAL.md

```
@@ -0,0 +1,700 @@
+```
+
+---
+
+## 提案4: JSON APIモード
+
+SPA / モダンフロントエンドとの統合を可能にします。
+
+### API仕様
+
+**エンドポイント**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)